package.json

```diff
@@ -12,8 +12,6 @@
         "build": "tsc",
         "lint": "tslint -p tsconfig.json -c tslint.json",
         "lint:fix": "tslint -p tsconfig.json -c tslint.json --fix",
-        "start": "rm -rf dist/ && tsc && node -r dotenv/config dist/main.js",
-        "start:debug": "nodemon --legacy-watch",
         "test": "NODE_ENV=LOCAL node -r dotenv/config node_modules/.bin/jest --runInBand",
         "test:local": "NODE_ENV=LOCAL node -r dotenv/config node_modules/.bin/jest --runInBand -- local",
         "test:integration": "node -r dotenv/config node_modules/.bin/jest --runInBand -- integration"
```

Comment:
@voutasaurus we could probably remove a lot more of these dependencies since we only use npm to run tests now... but I'm too lazy to figure out which ones to remove...

Reply:
is there a tool that will do that? like `npm clean` or something?